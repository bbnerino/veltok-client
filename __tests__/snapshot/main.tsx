import MainPage from "@/pages";
import { render } from "@testing-library/react";

it("renders correctly", () => {
  const { container } = render(<MainPage />);
  expect(container).toMatchSnapshot();
});
