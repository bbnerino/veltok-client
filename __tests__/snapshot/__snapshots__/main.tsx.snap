// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <main
    class="sc-bdDrbm cyiMOT"
  >
    <div
      data-aos="fade-up"
    >
      <img
        alt="red_spray"
        data-nimg="1"
        decoding="async"
        height="450"
        loading="lazy"
        src="/_next/image?url=%2Fimages%2Fred_spray.png&w=750&q=75"
        srcset="/_next/image?url=%2Fimages%2Fred_spray.png&w=384&q=75 1x, /_next/image?url=%2Fimages%2Fred_spray.png&w=750&q=75 2x"
        style="color: transparent;"
        width="350"
      />
    </div>
    <div
      data-aos="fade-right"
    >
      <img
        alt="logo"
        class="logo"
        data-nimg="1"
        decoding="async"
        height="100"
        loading="lazy"
        src="/_next/image?url=%2Fimages%2Flogo.png&w=750&q=75"
        srcset="/_next/image?url=%2Fimages%2Flogo.png&w=384&q=75 1x, /_next/image?url=%2Fimages%2Flogo.png&w=750&q=75 2x"
        style="color: transparent;"
        width="350"
      />
    </div>
  </main>
  <main
    class="sc-gtcAbF dXdqtC"
  >
    <div
      data-aos="fade-left"
    >
      <p
        class="text"
      >
        Grow with Veltok Communicate with friends and have fun
      </p>
    </div>
    <div
      data-aos="fade-right"
    >
      <img
        alt="rainbow_spray"
        class="rainbow_spray"
        data-nimg="1"
        decoding="async"
        height="450"
        loading="lazy"
        src="/_next/image?url=%2Fimages%2Frainbow_spray.png&w=1080&q=75"
        srcset="/_next/image?url=%2Fimages%2Frainbow_spray.png&w=640&q=75 1x, /_next/image?url=%2Fimages%2Frainbow_spray.png&w=1080&q=75 2x"
        style="color: transparent;"
        width="500"
      />
    </div>
    <div
      data-aos="fade-up"
    >
      <img
        alt="logo"
        class="logo"
        data-nimg="1"
        decoding="async"
        height="100"
        loading="lazy"
        src="/_next/image?url=%2Fimages%2Flogo.png&w=750&q=75"
        srcset="/_next/image?url=%2Fimages%2Flogo.png&w=384&q=75 1x, /_next/image?url=%2Fimages%2Flogo.png&w=750&q=75 2x"
        style="color: transparent;"
        width="350"
      />
    </div>
  </main>
  <main
    class="sc-dkXrac hxlVPQ"
  >
    <div
      data-aos="fade-up"
    >
      <p
        class="text text1"
      >
        Link with Github
      </p>
      <p
        class="sub_text sub_text1"
      >
        share with your friends
      </p>
      <div
        class="github_wrapper"
      >
        <img
          alt="github"
          data-nimg="1"
          decoding="async"
          height="200"
          loading="lazy"
          src="/_next/image?url=%2Fimages%2Fgithub.png&w=640&q=75"
          srcset="/_next/image?url=%2Fimages%2Fgithub.png&w=256&q=75 1x, /_next/image?url=%2Fimages%2Fgithub.png&w=640&q=75 2x"
          style="color: transparent;"
          width="200"
        />
      </div>
    </div>
    <div
      data-aos="fade-up"
    >
      <p
        class="text text2"
      >
        Community
      </p>
      <p
        class="sub_text sub_text2"
      >
        Communicate with Colleagues
      </p>
      <div
        class="community_wrapper"
      >
        <p
          class="big_text"
        >
          1000 +
        </p>
        <p
          class="sub_text"
        >
          people helped
        </p>
      </div>
    </div>
  </main>
  <div
    class="sc-eCQiwC frlbZ"
  >
    <div
      data-aos="fade-up"
    >
      <button
        class="sc-hKVryf ezbmRm"
      >
        Login
      </button>
      <button
        class="sc-hKVryf ezbmRm"
      >
        Join Now
      </button>
    </div>
  </div>
</div>
`;
